playbooks/roles/certificates/templates/openssl.cnf.j2

```diff
@@ -69,6 +82,7 @@ subjectKeyIdentifier=hash
 authorityKeyIdentifier=keyid,issuer:always
 extendedKeyUsage=serverAuth
 keyUsage = digitalSignature, keyEncipherment
+subjectAltName = @alt_names
 
 [ client ]
 basicConstraints=CA:FALSE
```

Comment:
I think the client certs also need an `extendedKeyUsage` that includes `clientAuth`. I _think_ that's why `playbooks/roles/openconnect/templates/client.tmpl.j2` had a line `tls_www_client` that I don't see replicated here.